package.json

```diff
@@ -75,5 +75,6 @@
         "rimraf": "^3.0.2",
         "styled-jsx": "^4.0.1",
         "wait-on": "^6.0.0"
-    }
+    },
+    "packageManager": "yarn@2.4.2"
```

Comment:
What's this for? If I recall correctly, I think we specifically want `yarn 1` for this repo

Reply:
I'm not even sure how that got in there.. I'm on a version of yarn 1.22 myself. I'll double check whats happening with the package.json files